pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
   sdk: '>=2.18.0 <3.0.0'
 
 dependencies:
+  collection: ^1.18.0
```

Comment:
NOTE: need to update open-source dependencies prior to merge (I'll take care of it, just marking it here so I don't forget)